compliance_checker/cf/cf.py

```diff
@@ -3432,6 +3621,53 @@ class CF1_7Check(CF1_6Check):
+                  'compress': {'Type': 'S', 'attr_loc': {'C'}, 'cf_section': '8.2'},
+                  'computed_standard_name': {'Type': 'S',
+                                             'attr_loc': {'C'},
+                                             'cf_section': '4.3.3'},
+                  'coordinates': {'Type': 'S', 'attr_loc': {'D'}, 'cf_section': '5'},
+                  'external_variables': {'Type': 'S', 'attr_loc': {'G'}, 'cf_section': '2.6.3'},
+                  'featureType': {'Type': 'S', 'attr_loc': {'G'}, 'cf_section': '9.4'},
+                  'flag_masks': {'Type': 'D', 'attr_loc': {'D'}, 'cf_section': '3.5'},
+                  'flag_meanings': {'Type': 'S', 'attr_loc': {'D'}, 'cf_section': '3.5'},
+                  'flag_values': {'Type': 'D', 'attr_loc': {'D'}, 'cf_section': '3.5'},
+                  'formula_terms': {'Type': 'S', 'attr_loc': {'C'}, 'cf_section': '4.3.3'},
+                  'grid_mapping': {'Type': 'S', 'attr_loc': {'D'}, 'cf_section': '5.6'},
+                  'history': {'Type': 'S', 'attr_loc': {'G'}, 'cf_section': None},
+                  #'instance_dimension': {'Type': 'S', 'attr_loc': {'-'}, 'cf_section': '9.3'},
+                  'institution': {'Type': 'S', 'attr_loc': {'G', 'D'}, 'cf_section': '2.6.2'},
+                  'leap_month': {'Type': 'N', 'attr_loc': {'C'}, 'cf_section': '4.4.1'},
+                  'leap_year': {'Type': 'N', 'attr_loc': {'C'}, 'cf_section': '4.4.1'},
+                  'long_name': {'Type': 'S', 'attr_loc': {'D', 'C'}, 'cf_section': '3.2'},
+                  'missing_value': {'Type': 'D', 'attr_loc': {'D', 'C'}, 'cf_section': '2.5.1'},
+                  'month_lengths': {'Type': 'N', 'attr_loc': {'C'}, 'cf_section': '4.4.1'},
+                  'positive': {'Type': 'S', 'attr_loc': {'C'}, 'cf_section': None},
+                  'references': {'Type': 'S', 'attr_loc': {'G', 'D'}, 'cf_section': '2.6.2'},
+                  #'sample_dimension': {'Type': 'S', 'attr_loc': {'-'}, 'cf_section': '9.3'},
+                  'scale_factor': {'Type': 'N', 'attr_loc': {'D', 'C'}, 'cf_section': '8.1'},
+                  'source': {'Type': 'S', 'attr_loc': {'G', 'D'}, 'cf_section': '2.6.2'},
+                  'standard_error_multiplier': {'Type': 'N',
+                                                'attr_loc': {'D'},
+                                                'cf_section': None},
+                  'standard_name': {'Type': 'S', 'attr_loc': {'D', 'C'}, 'cf_section': '3.3'},
+                  'title': {'Type': 'S', 'attr_loc': {'G'}, 'cf_section': None},
+                  'units': {'Type': 'S', 'attr_loc': {'D', 'C'}, 'cf_section': '3.1'},
+                  'valid_max': {'Type': 'N', 'attr_loc': {'D', 'C'}, 'cf_section': None},
+                  'valid_min': {'Type': 'N', 'attr_loc': {'D', 'C'}, 'cf_section': None},
+                  'valid_range': {'Type': 'N', 'attr_loc': {'D', 'C'}, 'cf_section': None}}
```

Comment:
Instead of copying a bunch of the 1.6 Appendix A, could you use

```
self.appendix_a.update({...})
```
with only the needed information? Or is that not feasible with this design?

Reply:
It's possible, yes.  Will DRY things up a bit.